monetio/sat/_tropomi_l2_no2_mm.py

```diff
@@ -54,45 +60,120 @@ def _open_one_dataset(fname, variable_dict):
     ds["scan_time"] = ds["time"] + dtime
     ds["scan_time"].attrs.update({"long_name": "scan time"})
     ds = ds.set_coords(["lon", "lat", "time", "scan_time"])
-    ds.attrs["reference_time_string"] = ref_time_val.astype(datetime).strftime(r"%Y%m%d")
+    ds.attrs["reference_time_string"] = ref_time_val.astype(datetime).strftime(r"%Y-%m-%d")
+
+    def get_extra(varname_, *, dct_=None, default_group="PRODUCT"):
+        """Get non-varname variables."""
+        if dct_ is None:
+            dct_ = variable_dict.get(varname_, {})
+        group_name = dct_.get("group", default_group)
+        if isinstance(group_name, list):
+            group_name = group_name[0]
+        return _get_values(dso[group_name][varname_], dct_)
+
+    for varname, dct in variable_dict.items():
+        print(f"- {varname}")
+        if dct is None:
```

Comment:
# from mli: the "None" is read as a str from yaml file, so cannot recognized
if (dct is None) or (dct == "None") or (dct == "NONE"):
            dct = {}

Reply:
According to the pyYaml docs (https://pyyaml.org/wiki/PyYAMLDocumentation), None should be represented either with ~ or with null

Reply:
Addressed in 483d088